_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -0,0 +1,204 @@
+      the highest-fee replacement after they've finished receiving their
+      new UTXO.  This won't earn them any money, but the high fees will
+      be deducted from the value of the honest party's UTXO and so there's no cost to
+      the attacker.
+
+    - *Costless pinning theft:* like above, if there are multiple
+      versions of the transaction with different feerates, an attacking
+      counterparty can broadcast a low-feerate version and then use a
+      [transaction pinning][topic transaction pinning] method to make it
+      impractical to fee bump.  If the attack is successful, it would
+      allow the attacker to spend the victim's money; if it was
+      unsuccessful, it might cost the attacker nothing extra if they
+      were already planning to send the transactions they used for
+      pinning (e.g. they were planning to consolidate UTXOs anyway).
+
+  Several potential solutions to these attacks were discussed.  The
+  [most promising solution][belcher collateral] was to require anyone
+  spending a coinswap UTXO to spend part of the value of one of their
+  other UTXOs to transaction fees.  This *collateral payment* would
+  eliminate the costless aspect of the previously described attacks.
+  Fully addressing the attacks might also require some other changes
+  related to timelocks, such as using a 1-block relative timelock ("`1
+  OP_CSV`") to prevent pinning.
+
+  In addition to the discussion helping to improve a privacy-enhancing
+  coinswap protocol, at least one participant [noted][zmnscpxj ln] that
+  some of the insights from the conversation might be applicable to
+  other protocols, such as LN.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Re-delegate absurd fee checking from mempool to clients][review club #19339] is
+a PR ([#19339][Bitcoin Core #19339]) by Gloria Zhao that proposes an updated
+version of [Bitcoin Core #15810][]. It adds separate fee-checking logic for
+clients that need to enforce max feerates and then removes the `nAbsurdFee`
+logic from `AcceptToMemoryPool`.
+
+The review club discussion covered general concepts before diving into the
+deeper technical aspects.
+
+{% include functions/details-list.md
+  q0="What does the `AcceptToMemoryPool` (ATMP) function in `validation.cpp` do?"
+  a0="ATMP is the function that Bitcoin Core calls to process a transaction
+      for potential acceptance to the mempool if it's acceptable. In addition to
+      enforcing consensus rules, ATMP also applies the node's policy."
+  a0link="https://bitcoincore.reviews/19339#l-55"
+
+  q1="What is \"policy\"?"
+  a1="Policy is generally defined as the network transaction rules and excludes
+      consensus rules and user/client preferences."
```

Comment:
This "excludes [...] user/client preferences" confuses me.  Many things from policy can be tweaked using bitcoin configuration parameters, so it seems implicit that policy can include user/client preferences.  (Additionally, Luke would likely be upset to hear someone say that policy isn't a user preference.)

I'm going to make an edit, but please feel free to revert if you don't like it.

Reply:
No worries, that part confuses me as well and wumpus, for instance, didn't say that in the quoted line or the following line.